lemonstand/lscloud-theme-meyer
# 267 multiple product options

- Removed $timeout to build query string parameters. $timeout was not necessary
- Code clean up and indentation 
- Fixed issue with options selector not showing default value or most recent value selected
- Fixed issue with options not getting added to cart properly
- Fixed issue with multiple options not even making it to cart

Test scenarios: 
- select the following product https://meyer-staging.lemonstand.com/product/ls-coder which has two options available.  Choose colour and size and then proceed to add to cart.  Result should show the colour and size selected
- Select any other product with single option such as this one https://meyer-staging.lemonstand.com/product/cartoon-lemons-red don't change value and check out.  Result should show default value in cart
- Repeat step above and select an option this time cart should show the newly selected option in cart.

<img width="1267" alt="screen shot 2017-10-16 at 11 44 53 pm" src="https://user-images.githubusercontent.com/24289426/31650791-d77f3d70-b2cd-11e7-9627-f76433bf6809.png">
